students/uss_tomasz/lesson_05_lists/5_3/bowling_alley.py

```diff
@@ -0,0 +1,26 @@
+from random import randint
+pins_quantity = 10
+balls_quantity = 5
```

Comment:
these 2 are constants and should be named according to PEP8 https://www.python.org/dev/peps/pep-0008/#constants